presto-tpcds/src/main/resources/META-INF.services/com.facebook.presto.spi.Plugin

```diff
@@ -0,0 +1 @@
+com.facebook.presto.tpcds.TpcdsPlugin
```

Comment:
I don't know how exactly it works, it's maybe part of the provisio plugin? In any case, none of the other connectors have this file and they work properly; maybe tpch had remnants of an older system.